Domain/Academy/src/main/java/tw/waterball/judgegirl/academy/domain/usecases/exam/CreateExamTranscriptUseCase.java

```diff
@@ -72,16 +74,14 @@ public void execute(int examId, Presenter presenter) {
                 Record::getSubmissionId).toArray(new String[0]);
     }
 
-    private GetById<Integer, Student> examinees(Exam exam) {
-        var idToExaminee = toMap(studentServiceDriver.getStudentsByIds(
+    private Map<Integer, Student> examinees(Exam exam) {
+        return toMap(studentServiceDriver.getStudentsByIds(
                 mapToList(exam.getExaminees(), Examinee::getStudentId)), Student::getId, identity());
-        return idToExaminee::get;
     }
 
-    private List<ExamineeRecord> examineeRecords(GetById<Integer, Student> examinees, List<QuestionRecord> records) {
+    private List<ExamineeRecord> examineeRecords(Map<Integer, Student> examinees, List<QuestionRecord> records) {
```

Comment:
NIT: Why do you change ```GetById``` to ```Map```?

Reply:
@snowmancc WHY?

Reply:
I need to set the student list into the presenter, So I need the map to use them achieved goal~

Reply:
got it